### ISSUE TITLE
✨ Add  "organization, platform, or theme" field to Personal experiences

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/gc-digital-talent#12329]
## ✨ Feature

Add a new field to Personal experiences title "Organization, platform, or theme".

## 🕵️ Details

The intention is that this field would allow related Personal experiences to be grouped together. It would be equivalent to the Organization fields in other experience types, and autopopulated in the same way.

## 🎨 Design

@JoshBeveridge 

## 📸 Screenshot

![Image](https://github.com/user-attachments/assets/e098ddfb-cef3-4f61-9e97-21cd5ba53e4d)

## ✅ Acceptance Criteria

- [ ] new field added to database, schema, factories and seeders
- [ ] new field added to Personal experience form
  - [ ] comes with additional context
  - [ ] autopopulated like other Experience Org fields (sharing the same list of orgs with other experiences)

## 🛑 Blockers

```[tasklist]
### Blocked By
```
https://www.figma.com/design/tam2tiCQQeZoGiQ9jMM58c/Career-experience-(Applicants)?node-id=64-78760&t=bmG0h2aMdhnfqbvW-0

**Comment by gobyrne** [comment]
This is low priority. ice-boxing

**Comment by petertgiles** [comment]
No movement.  Reopen if we want to start working on it.
